homeassistant/components/incomfort/climate.py

```diff
@@ -22,6 +22,8 @@
 from .coordinator import InComfortDataCoordinator
 from .entity import IncomfortEntity
 
+PARALLEL_UPDATES = 0
```

Comment:
For climate/water_heater keep in mind that that also means that services can be executed at the same time

Reply:
Good point updated the climate platform to `1`.